### DISCUSSION TITLE
Sending commit notifications to collaborators

### DISCUSSION THREAD

**Discussion by lambrtz** [community/community#22619] [New to GitHub]
<p>I am managing private repositories in an organization account and would like to notify our collaborators (not team members) e.g. by emails whenever a commit is done. Would it be possible to do this in GitHub? My search returned this thread:</p>
<p><a href="https://github.community/t/get-notified-of-all-commits-to-all-branches-of-a-repo/759">https://github.community/t5/How-to-use-Git-and-GitHub/Get-notified-of-ALL-commits-to-ALL-branches-of-a-repo/m-p/5256</a></p>
<p>that suggested to set up a webhook. I was wondering if there is a simpler way to do this.</p>
<p>Thank you.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey there <a class="mention" href="/lambrtz">@lambrtz</a>, help’s arrived!</p>
<p>GitHub doesn’t support this right now, but the teams at GitHub are always on the look out on the forums and I’m sure they’ll be able to pass on your feedback. Otherwise, you can always reach out to the folks at GitHub support here and let them know you’d like to see this: <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> 🙂</p>
<p>Let me know if you need a hand with anything else.</p>
